bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -436,22 +371,18 @@ async def _edit_nomination_reason(
             else:
                 raise
 
-        if not nomination["active"]:
-            await ctx.send(f":x: <@{nomination['user']}> doesn't have an active nomination.")
+        if not nomination.active:
```

Comment:
Question: Will this path ever be executed ?
Since we now pass `active` as `True` when bringing nominations

Reply:
If `target` is an `int` then it gets the nomination straight from that and it can be inactive, rather than if a member/user is passed in which case it does that search for active nominations only.

Reply:
Ah I see now, thanks !

Reply:
I noticed that bit and another place could be refactored a bit to make it a bit neater and remove a potential API call, so did that.